school/models/student.py

```diff
@@ -244,7 +244,23 @@ def check_current_year(self):
                                      compute="_compute_teacher_user",
                                      )
     active = fields.Boolean(default=True)
-
+    
+    # image: all image fields are base64 encoded and PIL-supported
+    image = fields.Binary(
+        "Photo", default=_default_image, attachment=True,
+        help="This field holds the image used as photo for the employee, limited to 1024x1024px.")
+    image_medium = fields.Binary(
```

Comment:
**image_small** and **image_medium** should be **compute** fields which will resize the image uploaded in **photo** field.